RecoTracker/TkSeedingLayers/python/DisplacedGeneralLayerTriplet_cfi.py

```diff
@@ -0,0 +1,130 @@
+    'TID2+TID3+TEC1_pos', #'TID2+TID3+TEC1_neg',
+    'TID2+MTID3+TEC1_pos', 'TID2+MTID3+TEC1_neg',
+    'MTID3+TEC1_pos+MTEC2_pos', 'MTID3+TEC1_neg+MTEC2_neg'
+
+)
+
+
+
+
+
+DisplacedGeneralLayerTriplet.TOB = cms.PSet(
+        matchedRecHits = cms.InputTag("siStripMatchedRecHits","matchedRecHit"),
+        TTRHBuilder = cms.string('WithTrackAngle'),
+        clusterChargeCut = cms.PSet(refToPSet_ = cms.string('SiStripClusterChargeCutNone')),
+        skipClusters   = cms.InputTag('displacedGeneralStepClusters')
+)
+
+
+DisplacedGeneralLayerTriplet.MTOB = cms.PSet(
+        TTRHBuilder = cms.string('WithTrackAngle'),
+        rphiRecHits    = cms.InputTag("siStripMatchedRecHits","rphiRecHit"),
+        clusterChargeCut = cms.PSet(refToPSet_ = cms.string('SiStripClusterChargeCutNone')),
+        skipClusters   = cms.InputTag('displacedGeneralStepClusters')
+)
+
+
+DisplacedGeneralLayerTriplet.TIB = cms.PSet(
+        TTRHBuilder    = cms.string('WithTrackAngle'), 
+	clusterChargeCut = cms.PSet(refToPSet_ = cms.string('SiStripClusterChargeCutNone')),
+        matchedRecHits = cms.InputTag("siStripMatchedRecHits","matchedRecHit"),
+        skipClusters   = cms.InputTag('displacedGeneralStepClusters')
+)
+
+DisplacedGeneralLayerTriplet.MTIB = cms.PSet(
+        TTRHBuilder    = cms.string('WithTrackAngle'), 
+	clusterChargeCut = cms.PSet(refToPSet_ = cms.string('SiStripClusterChargeCutNone')),
+        rphiRecHits    = cms.InputTag("siStripMatchedRecHits","rphiRecHit"),
+        skipClusters   = cms.InputTag('displacedGeneralStepClusters')
+)
+
+DisplacedGeneralLayerTriplet.TID = cms.PSet(
+        matchedRLayerecHits = cms.InputTag("siStripMatchedRecHits","matchedRecHit"),
+        useRingSlector = cms.bool(True),
+        TTRHBuilder = cms.string('WithTrackAngle'), 
+	clusterChargeCut = cms.PSet(refToPSet_ = cms.string('SiStripClusterChargeCutNone')),
+        minRing = cms.int32(1),
+        maxRing = cms.int32(2),
+        skipClusters   = cms.InputTag('displacedGeneralStepClusters')
+
+)
+
+DisplacedGeneralLayerTriplet.MTID = cms.PSet(
+        rphiRecHLayerits    = cms.InputTag("siStripMatchedRecHits","rphiRecHit"),
+        useRingSlector = cms.bool(True),
+        TTRHBuilder = cms.string('WithTrackAngle'), 
+        clusterChargeCut = cms.PSet(refToPSet_ = cms.string('SiStripClusterChargeCutNone')),
+        minRing = cms.int32(3),
+        maxRing = cms.int32(3),
+        skipClusters   = cms.InputTag('displacedGeneralStepClusters')
+
+)
+
+DisplacedGeneralLayerTriplet.TEC = cms.PSet(
+        matchedRLayerecHits = cms.InputTag("siStripMatchedRecHits","matchedRecHit"),
+        useRingSlector = cms.bool(True),
+        TTRHBuilder = cms.string('WithTrackAngle'), 
+        clusterChargeCut = cms.PSet(refToPSet_ = cms.string('SiStripClusterChargeCutNone')),
+        minRing = cms.int32(5),
+        maxRing = cms.int32(5),
+        skipClusters   = cms.InputTag('displacedGeneralStepClusters')
+
+)
```

Comment:
```suggestion
DisplacedGeneralLayerTriplet.TEC = DisplacedGeneralLayerTriplet.TID.clone(
        minRing = 5,
        maxRing = 5,
)
```

follow a similar pattern in other cases: less copy-paste